docs/topics/gradle/gradle-compilation-and-caches.md

```diff
@@ -291,6 +291,26 @@ When configuring the Kotlin daemon's JVM arguments, note that:
   {type="note"}
 * If the `Xmx` argument is not specified, the Kotlin daemon will inherit it from the Gradle daemon.
 
+## The new Kotlin compiler
+
+The new Kotlin K2 compiler is in [Alpha](components-stability.md#stability-levels-explained).
+It has basic support for Kotlin JVM, JS, and Native projects.
+
+The new compiler aims to speed up the development of new language features, unify all of the platforms Kotlin supports,
+bring performance improvements, and provide an API for compiler extensions.
+
+The K2 compiler will become the default starting with Kotlin 2.0. To try it in your projects now and check the performance,
+use the `kotlin.experimental.tryK2=true` Gradle property or run the following command:
+
+```shell
+./gradlew assemble -Pkotlin.experimental.tryK2=true
+```
+
+This Gradle property automatically sets the language version to 2.0 and updates the [build report](#build-reports)
```

Comment:
This property does not override explicitly configured language version, so better highlight that it configured default one:
```suggestion
This Gradle property automatically sets default language version to 2.0 and updates the [build report](#build-reports)
```